packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -100,6 +111,7 @@ const getNavigationCommandLoaderPerPostType = ( postType ) =>
 				};
 
 				if (
+					! canCreateTemplate ||
 					postType === 'post' ||
 					( postType === 'page' && ! isBlockBasedTheme )
```

Comment:
Why do we need the custom postType checks here? I know it's pre-existent but maybe we can remove this with the canCreateCheck (if it relies on post type)

Reply:
I think this is because the site editor only has the `page` post type screen yet - if the post type is `post`, we need to link to the post editor and not the site editor, even if it's a block theme or admin user.